librz/include/rz_bin.h

```diff
@@ -281,8 +281,10 @@ typedef struct rz_bin_object_t {
 	RzBinRelocStorage *relocs;
 	RzBinStrDb *strings;
 	RzList /*<RzBinClass *>*/ *classes;
-	HtPP *classes_ht;
-	HtPP *methods_ht;
+	HtPP *name_to_class_object; ///< map[char *]RzBinClass
+	HtPP *glue_to_class_method; ///< map[char *]RzBinSymbol
+	HtPP *glue_to_class_field; ///< map[char *]RzBinClassField
+	HtUP *vaddr_to_class_method; ///< map[ut64]RzBinSymbol
```

Comment:
i think we use a sliughtly different syntax like `HtUP /*<offset, List *<LocListEntry>*/ *loc_table`